src/net_processing.cpp

```diff
@@ -57,8 +57,8 @@ static constexpr auto HEADERS_DOWNLOAD_TIMEOUT_PER_HEADER = 1ms;
  * behind headers chain.
  */
 static constexpr int32_t MAX_OUTBOUND_PEERS_TO_PROTECT_FROM_DISCONNECT = 4;
-/** Timeout for (unprotected) outbound peers to sync to our chainwork, in seconds */
-static constexpr int64_t CHAIN_SYNC_TIMEOUT = 20 * 60; // 20 minutes
+/** Timeout for (unprotected) outbound peers to sync to our chainwork */
```

Comment:
Will also need to remove "seconds" from where `CHAIN_SYNC_TIMEOUT` is used in docs?

Reply:
Just to confirm. Are you talking about this line?

https://github.com/bitcoin/bitcoin/blob/d69af93223c4008c3255f7e4848ff05d78c514fa/src/net_processing.cpp#L729-L730

Reply:
Yes

Reply:
This line could use a fixing.

However, doing so will revoke the current 3 ACKs on the current top commit of this PR.
So would you suggest I should update this PR for this change?

Reply:
If it is fixed, it should be fixed here to avoid another follow-up pull. If not, it might be best to leave it as-is.

A re-ACK is less overhead and effort than a new pull + fresh ACK.

Reply:
Fixed in fe86eb50c986f7b5ccce63f984d8a51cd9ee9e2c